cypress/plugins/synapsedocker/templates/COPYME/homeserver.yaml

```diff
@@ -12,7 +12,7 @@ listeners:
     x_forwarded: true
 
     resources:
-      - names: [client, federation, consent]
```

Comment:
Removing from the copyme template because it's not necessary in the base config to copy from.

Additonally, `user_consent` isn't configured here and it will complain.